packages/components-native/src/Divider/DividerHorizontal.style.ts

```diff
@@ -1,17 +1,22 @@
 import { StyleSheet } from "react-native";
 import { tokens } from "../utils/design";
 
-export const styles = StyleSheet.create({
+export const horizontalStyles = StyleSheet.create({
   base: {
     height: tokens["space-minuscule"],
     margin: 0,
-    borderBottomWidth: tokens["border-base"],
+    width: "auto",
     borderBottomColor: tokens["color-border"],
+    borderBottomWidth: tokens["space-minuscule"],
```

Comment:
This seems to be fairly common throughout the codebase and I did the same myself but Chris [recently pointed out](https://github.com/GetJobber/atlantis/pull/2096#discussion_r1829960573) that space values should not be used to widths. There are a couple of instances of this in these two styles files

Reply:
yeahhh that's fair - updated!